app/policies/participant_profile_policy.rb

```diff
@@ -1,6 +1,16 @@
 # frozen_string_literal: true
 
 class ParticipantProfilePolicy < ApplicationPolicy
+  def show?
+    admin_only
+  end
+
+  def destroy?
+    admin? && (record.ect? || record.mentor?)
```

Comment:
It's not obvious to me why the restriction to ect and mentor is needed. Perhaps a code comment to explain?

Reply:
That brings up another interesting point. Will we show the link to delete the user on purely npq users that are visible from participants page? I think we will :(

Reply:
Currently there are no links to the show page for an NPQ participant, and if you try to visit it by entering the ID then it throws and error about not having a cohort. That said, I should hide it for when it does work 🙂

Reply:
Now done